planning/behavior_path_planner/autoware_behavior_path_avoidance_by_lane_change_module/src/interface.cpp

```diff
@@ -47,10 +47,6 @@ bool AvoidanceByLaneChangeInterface::isExecutionRequested() const
   return module_type_->isLaneChangeRequired() && module_type_->specialRequiredCheck() &&
          module_type_->isValidPath();
 }
-void AvoidanceByLaneChangeInterface::processOnEntry()
```

Comment:
@kobayu858 Thank you for your work. But this function will be called in here.
https://github.com/autowarefoundation/autoware.universe/blob/602e3d068afb8b0543e579a30683a938331c3d2b/planning/behavior_path_planner/autoware_behavior_path_planner_common/include/autoware/behavior_path_planner_common/interface/scene_module_interface.hpp#L182